packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -385,6 +385,13 @@ export interface ClusterProps {
    * @default - false
    */
   readonly enhancedVpcRouting?: boolean;
+
+  /**
+   * Indicating whether Amazon Redshift should deploy the cluster in two Availability Zones.
+   *
+   * @default false
```

Comment:
I thought this would be more appropriate since we are not actually setting false in CDK.

```suggestion
   * @default - false
```